src/frontend/shortlist/src/api/shortlist.js

```diff
@@ -36,10 +36,37 @@ export default class ShortlistApi {
   authLogin(payload, successCb, failCb) {
     return new tempAuthLogin(payload, successCb, failCb);
   }
+  removeSchoolCard(payload, successCb, failCb) {
+    return new tempRemoveSchoolCard(payload, successCb, failCb);
+  }
 }
 
 import axios from "axios";
 import md5 from "md5";
+
+export class tempRemoveSchoolCard {
+  constructor(payload, successCb, failCb) {
+    this.recoID = payload.recoID;
+    this.accepted = payload.accepted;
+    this.successCb = successCb;
+    this.failCb = failCb;
+  }
+  execute() {
+    // Send the accepted card first
+    axios({
+      method: "POST",
+      url: "https://api.shortlist.nyc/recommendation/update",
+      headers: {},
+      data: {
```

Comment:
```suggestion
      data: payload
```

Reply:
Updated

Reply:
I think this is still the same - lines 61 and 62? And if you change to using payload directly make sure to define it in the constructor above